vite.config.ts

```diff
@@ -0,0 +1,42 @@
+import react from "@vitejs/plugin-react-swc";
+import { defineConfig } from "vitest/config";
+import chokidar from 'chokidar';
+import path from 'path';
+
+export default defineConfig({
+	plugins: [
+		react(),
+		{
+			name: 'watch-for-svg-changes',
+			configureServer(server) {
+				let processSVGs;
+				if (typeof window === 'undefined') {
+					processSVGs = require('./tools/process-svgs.js').processSVGs;
+				}
+
+				const delayMs = 1_000;
+				let timeoutId: NodeJS.Timeout | undefined = undefined;
+
+				chokidar
+					.watch([path.join(__dirname, "svgs")], {
+						ignoreInitial: false,
+					})
+					.on("all", (event, filePath) => {
+						console.log(`Detected change in SVGs: ${event} ${filePath}`);
```

Comment:
This and the other console.log should only run when a file is actually changed, not on the initial build. Otherwise it fills the terminal and you can't even see the link from Vite to the dev page!